pkgs/development/python-modules/pycdio/default.nix

```diff
@@ -29,23 +32,33 @@ buildPythonPackage rec {
     patchShebangs .
   '';
 
+  build-system = [ setuptools ];
+
   nativeBuildInputs = [
-    nose
-    pkgs.pkg-config
-    pkgs.swig
+    pkg-config
+    swig
   ];
-  buildInputs = [
-    setuptools
-    pkgs.libcdio
-  ] ++ lib.optional stdenv.isDarwin pkgs.libiconv;
+
+  dependencies = [ libcdio ] ++ lib.optional stdenv.isDarwin libiconv;
+
+  nativeCheckInputs = [ pytestCheckHook ];
 
   # Run tests using nosetests but first need to install the binaries
   # to the root source directory where they can be found.
-  checkPhase = ''
+  preCheck = ''
     ./setup.py install_lib -d .
-    nosetests
+
+    substituteInPlace test/test-cdtext.py \
+      --replace-fail "assertEquals" "assertEqual"
   '';
 
+  pytestFlagsArray = [
```

Comment:
Maybe we can rename them to make pytest find them on itself?

Reply:
I'm more in favor to keep them in `pytestFlagsArray` rather than trying to patch the filenames